File: ServiceSuiKi/DomainModel/Personne.cs
using System.Net.Mail;

namespace DomainModel
{
    public class Personne
    {
        public int Id_Perso { get; set; }
        public int Id_User { get; set; }
        public string Nom_Personne { get; set; }
        public string Prenom_Personne { get; set; }
        public MailAddress Mail { get; set; }
        public string Patient_Medecin { get; set; }
    }
}

File: Suiki/app/src/main/java/com/suiki/suiki/Model/Personne.java
package com.suiki.suiki.Model;

import com.google.gson.annotations.SerializedName;

import org.simpleframework.xml.Element;

/**
 * Created by Massil on 11/01/2017.
 */

public class Personne {
    @SerializedName("ID_PERSONNE")
    public int ID_PERSONNE ;
    @SerializedName("ID_USER")
    public int ID_USER ;
    @SerializedName("NOM_PERSONNE")
    public String NOM_PERSONNE ;
    @SerializedName("PRENOM_PERSONNE")
    public String PRENOM_PERSONNE ;
    @SerializedName("MAIL")
    public String MAIL ;
    @SerializedName("PATIENT_MEDECIN")
    public String PATIENT_MEDECIN ;
}
